src/chains/ethereum/ethereum/src/helpers/common-helpers.ts

```diff
@@ -0,0 +1,42 @@
+import type { Common } from "@ethereumjs/common";
+
+/**
+ * Effectively removes the 3860 InitCode Size Limit check by increasing the init
+ * code size to Number.MAX_SAFE_INTEGER.
+ *
+ * This number works because an initcode can practically never be that large
+ * (approx 9 PetaBytes!).
+ *
+ * @param common
+ * @returns
+ */
+export function removeEIP3860InitCodeSizeLimitCheck(common: Common) {
```

Comment:
this is a hack until EJS ships a `allowUnlimitedInitCodeSize` option.

Reply:
Worth a comment?
```suggestion
export function removeEIP3860InitCodeSizeLimitCheck(common: Common) {
  // this is a hack until EJS ships `allowUnlimitedInitCodeSize` option https://github.com/ethereumjs/ethereumjs-monorepo/issues/2588
```

Reply:
Also, looks like they've settled on `allowUnlimitedInitcodeSize` lowercase `code` - do we want to match?

Reply:
They switched to `allowUnlimitedInitCodeSize`: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2597